wordpress-mobile/WordPress-iOS
# Update project to recommended XCode 9 settings

**Fixes** #7900 
Mostly by adding missing voids to comply with the strict prototype policy.

**To test:**
1. Clean + Build, check that there are no warnings other than: 
1.1 Convesion to Swift 4 is available.
1.2 Case is already handled by previous patterns; consider removing it (this one looks like a bug, @nheagy already looked into it, but if you want to double check is much appreciated)
1.3 Our old friend: Some object files have incompatible Objective-C category definitions....
2. Run the app, check that everything works as expected.

Needs review: @koke
